import_researchers.py

```diff
@@ -3,18 +3,15 @@
 
 from codecs import open
 
-from backend.create import *
-from papers.models import *
+from papers.models import Researcher, Department
```

Comment:
Imports can be sorted.

*PyImportSortBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/import_researchers.py
+++ b/import_researchers.py
@@ -3,7 +3,8 @@
 
 from codecs import open
 
-from papers.models import Researcher, Department
+from papers.models import Department
+from papers.models import Researcher
 from papers.name import normalize_name_words
 
 last_name_f = 0
```